esp/platform/esp.hpp

```diff
@@ -235,15 +235,13 @@ interface IEspContext : extends IInterface
     virtual void setRequest(IHttpMessage* req) = 0;
     virtual IHttpMessage* queryRequest() = 0;
 
-    virtual void setActiveSpan(ISpan * span)=0; // Only call this function directly if this object's lifetime matches the lifetime of the span.  If there is any doubt use EspContextSpanScope(ctx, span)
+    virtual void setServerSpan(ISpan * span)=0;     // Call this function to set the server span for the query.  The spans's lifetime will match the lifetime of the context object.
```

Comment:
I would consider renaming to setRequestSpan, it's a server span, but we want it to match the request's lifespan.